ocaml/xapi/xapi_pool_update.ml

```diff
@@ -291,7 +291,7 @@ let attach_helper ~__context ~uuid ~vdi ~use_localhost_proxy =
             "/dev/" ^ Client.VBD.get_device ~rpc ~session_id ~self:vbd
         )
       in
-      with_api_errors (mount device) mount_point ;
+      with_api_errors (mount ~ty:(Some "iso9660") device) mount_point ;
```

Comment:
Not against, but what the advantage of doing this? A different error in some cases? What if in the future we want to change file system?

Reply:
This encodes our expectations. Any other format should be rejected; if we want to change the format we will change the code.